aries_cloudagent/wallet/routes.py

```diff
@@ -355,13 +355,20 @@ async def wallet_create_did(request: web.BaseRequest):
                 f" support key type {key_type.key_type}"
             )
         )
+    seed = None
```

Comment:
The seed property is not added to the `DIDCreateSchema` making it harder to discover from the swagger ui and it won't be included when generating clients from the swagger file.

Reply:
@ianco your thoughts on this?

Reply:
agreed